.gitignore

```diff
@@ -49,6 +49,8 @@ docker-compose.local.yaml
 
 ## Project configurations
 .env
+.env.dev
+.env.testing
```

Comment:
Well, actually I don't really understand for what do we need this.

Could you please add any kind of examples to the README file?
I mean why should we use different .env files in general? How are you going to export them into the environment variables?

Reply:
READ.me
<img width="599" alt="image" src="https://user-images.githubusercontent.com/17402086/206463037-c94ac763-3003-4d4e-8180-c0b24f3d7a4c.png">
<img width="590" alt="image" src="https://user-images.githubusercontent.com/17402086/206463108-8fce9656-e949-4a02-bd27-f8c4cf0a946e.png">
You have to do some manual jobs to run pytest or run alembic, in my case if you create environments with your own configurations, development and testing environment works independently.